src/test/regress/input/directory_table.source

```diff
@@ -0,0 +1,565 @@
+CREATE STORAGE SERVER oss_server1;
+CREATE STORAGE SERVER oss_server2 OPTIONS(protocal 'localhost');
+CREATE STORAGE SERVER oss_server3 OPTIONS(endpoint '127.0.0.1:9000');
+CREATE STORAGE SERVER oss_server4 OPTIONS(https 'true');
+CREATE STORAGE SERVER oss_server5 OPTIONS(virtual_host 'false');
+CREATE STORAGE SERVER oss_server6 OPTIONS(protocol 'qingstor', endpoint 'pek3b,qingstor.com');
+CREATE STORAGE SERVER oss_server7 OPTIONS(https 'false', virtual_host 'true');
+CREATE STORAGE SERVER oss_server8 OPTIONS(protocol 'hdfs', namenode '127.0.0.1:8020');
+CREATE STORAGE SERVER oss_server9 OWNER TO postgres;    -- fail
+CREATE STORAGE SERVER IF NOT EXISTS oss_server10;
+CREATE STORAGE SERVER IF NOT EXISTS oss_server11 OPTIONS(protocol 's3av2');
+CREATE STORAGE SERVER IF NOT EXISTS oss_server12 OPTIONS(protocol 's3av2', endpoint '127.0.0.1:9000', https 'false');
+CREATE STORAGE SERVER IF NOT EXISTS oss_server13 OWNER TO postgres;   -- fail
+
+SELECT srvname, srvacl, srvoptions from gp_storage_server ORDER BY 1;
+\c dirtable_db
+SELECT srvname, srvacl, srvoptions from gp_storage_server ORDER BY 1;
+\c regression
+
+-- Test ALTER STORAGE SERVER
+ALTER STORAGE SERVER oss_server1 OPTIONS(protocol 'aws');
+ALTER STORAGE SERVER oss_server1 OPTIONS(protocol 'test');  -- fail
+ALTER STORAGE SERVER oss_server2 OPTIONS(https 'true');
+ALTER STORAGE SERVER oss_server2 OPTIONS(https 'false', virtual_host 'true');   -- fail
+ALTER STORAGE SERVER oss_server2 OPTIONS(virtual_host 'true');
+ALTER STORAGE SERVER oss_server3 OPTIONS(endpoint '192.168.0.1');   -- fail
+ALTER STORAGE SERVER oss_server4 OPTIONS(protocol 'localhost', virtual_host 'true');
+ALTER STORAGE SERVER oss_server4 OPTIONS(protocol 'qingstor');  -- fail
+ALTER STORAGE SERVER oss_server5;   -- fail
+ALTER STORAGE SERVER oss_server6 OWNER TO postgres; -- fail
+ALTER STORAGE SERVER IF EXISTS oss_server7 OPTIONS(endpoint '127.0.0.1:6555');    -- fail
+ALTER STORAGE SERVER IF NOT EXISTS oss_server8 OPTIONS(virtual_host 'true');    -- fail
+
+SELECT srvname, srvacl, srvoptions from gp_storage_server ORDER BY 1;
+\c dirtable_db
+SELECT srvname, srvacl, srvoptions from gp_storage_server ORDER BY 1;
+\c regression
+
+-- Test CREATE STORAGE USER MAPPING
+CREATE STORAGE USER MAPPING FOR CURRENT_USER;   -- fail
+
+CREATE STORAGE USER MAPPING FOR CURRENT_USER STORAGE SERVER oss_server1;
+
+CREATE STORAGE USER MAPPING FOR CURRENT_USER STORAGE SERVER oss_server1
+OPTIONS (accesskey 'KGFQWEFQQEFXVAEAWLLC', secretkey '0SJIWiIATh6jOlmAKr8DGq6hOAGBI1BnsnvgJmTs');   -- fail
+
+CREATE STORAGE USER MAPPING IF NOT EXISTS FOR CURRENT_USER STORAGE SERVER oss_server1;
+
+CREATE STORAGE USER MAPPING IF NOT EXISTS FOR CURRENT_USER STORAGE SERVER oss_server1
+OPTIONS (auth_method 'simple');
+
+CREATE STORAGE USER MAPPING FOR CURRENT_ROLE STORAGE SERVER oss_server2
+OPTIONS (accesskey 'EQEJIOJFAKQWESQEJIWQ', secretkey '0ADQiAxcaUJ2lMHipis80hsUEhdiqui82JhduOKE');
```

Comment:
export accesskey, secretkey in public ?

Reply:
Yes, now it is in public, maybe we will encrypt that in the future.